docusaurus.config.js

```diff
@@ -79,6 +79,24 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      metadata: [
+        {
+          property: "og:image",
+          content: "https://botkube.io/images/botkube-title-social.png",
```

Comment:
@madebyrogal 
We should use the `themeConfig.image` property instead, to support both `og:image` and `twitter:image` and probably others as well:
https://docusaurus.io/docs/api/themes/configuration#meta-image